lib/menu.rb

```diff
@@ -0,0 +1,27 @@
+class Menu
+attr_reader :dishes
+
+  def initailize(dishes)
+   @dishes = dishes
+  end
+
+#   def dishes
+#   dishes = 
+#         {
+#               "Margherita Pizza": 8.99},
+#       #   {name: "Salami Pizza", price: 9.99},
+#       #   {name: "Hawaiian Pizza", price: 9.99},
+#       #   {name: "Chips", price: 4.99},
+#   }
+#   end
+
+  def print_dishes
```

Comment:
I like the way you kept these methods in the separate classes and called them in the takeaway class. I think I should have done something more similar to this.